optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11.md

```diff
@@ -23,3 +23,9 @@ Optimize now supports Elasticsearch `8.7`.
 See the [supported environments]($docs$/reference/supported-environments) section for the full range of supported versions.
 
 If you need to update your Elasticsearch cluster, refer to the general [Elasticsearch update guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html). Usually, the only thing you need to do is perform a [rolling update](https://www.elastic.co/guide/en/elasticsearch/reference/current/rolling-upgrades.html).
+
+## Changes in the configuration
+
+Prior to Optimize 3.11.0, Optimize has performed collection role cleanup after syncing identities with the engine. From
+Optimize 3.11.0 onwards, this is now disabled by default. It can be reenabled by setting the
+`import.identitySync.collectionRoleCleanupEnabled` property value to `true`
```

Comment:
The setting is set to `false` for the pre-3.11 example config files as well in this PR - are those supposed to be set to `true`?

Reply:
good point, fixed 👍